src/controllers/accommodation.controller.js

```diff
@@ -85,3 +85,40 @@ export async function deleteAccommodation(req, res) {
     return res.status(400).send(err);
   }
 }
+
+export async function likeOrUnlikeAccommodation(req, res) {
+  try {
+    const userId = req.params.userId;
```

Comment:
We should not get the userId from the controller ==> we do not need to add any UserId from the input because as the user is autheticated, his/her id is already in the request (after authcheck.)